Makefile

```diff
@@ -267,69 +257,53 @@ pylint:
 new-service:
 	.venv/bin/cookiecutter gh:itisfoundation/cookiecutter-simcore-pyservice --output-dir $(CURDIR)/services
 
+# cache ------------------------------------------------------------------------------------
+.PHONY: pull-cache
+pull-cache:
```

Comment:
Should add help as well to the new makefile commands (and then you can check doc in the checklist)
For instance, 
```bash
# target: pull-cache – pulls all cache images from dockerhub registry
```
then you can ``make help`` and will list the target with a short help

Same applies to the others

Reply:
Done together in my PR